Library/Formula/postgresql.rb

```diff
@@ -82,6 +93,15 @@ def install
 
     system "./configure", *args
     system "make", "install-world"
+
+    # Now build and install pgroonga as optional resource
+    if build.with? "pgroonga"
+      resource("pgroonga").stage do
+        ENV.append_path "PATH", Formula["postgresql"].bin
```

Comment:
It should be just `bin` here. You don't need `Formula["postgresql"]` to point itself.